Source/SelfService/Web/api/api.ts

```diff
@@ -156,9 +157,9 @@ export async function deleteMicroservice(applicationId: string, environment: str
     return result.status === 200;
 }
 
-export async function saveMicroservice(input: any): Promise<boolean> {
+export async function saveMicroservice(input: any): Promise<any> {
```

Comment:
Don't we have a type for the response in this case?

Reply:
There's no generic represenation of the returned data from platform-api. `MicroserviceSimple` with the `MicroserviceSimpleExtra` types are mostly there, but they don't have all of the properties that we get back for a POAPI creation, like `webhooks` or `writeTo`.

Reply:
I could make it into an `object` though

Reply:
Also just as an observation, right now the response isn't being used for anything except showing the potential error message. It's just being passed through a few different functions until it lands in the original caller `_onSave` in `container.tsx` and it's just discarded.

Reply:
Ok, an object doesn't really help. I thought we had a matching TS declaration of the type - but then we can leave it for when we need the output later.